### PR TITLE
Feat : Home Page

### PR DESCRIPTION
**Pull request by manojkumar8400**
Added home page, page not found component and routes
Screenshot from 2022-04-28 23-19-42
Screenshot from 2022-04-28 23-19-45
.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,32 @@
...
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.app{
+  font-family: 'bad', cursive;
 }
-
-.App-link {
-  color: #61dafb;
+a{
```
Comment (srishti-maurya) [inline src/App.css:21]: you may avoid directly styling the tag

Comment (srishti-maurya) [reply src/App.css:21]: similarly with `ul` and `button` you may give it a class instead

**Review by srishti-maurya — APPROVED** [review]
looks good

**Review by bipin7yadav — APPROVED** [review]
Avoid styling tags . Else good to go.

**Pull request merged**